hdf/examples/AN_create_annotation.c

```diff
@@ -1,5 +1,11 @@
 #include "hdf.h"
 
+/* Used to make certain a return value _is_not_ a value.  If not ture, */
```

Comment:
Is "ture" a misspelling of true?  Also, it looks like CHECK(ret, val, where) is checking to see that the return value is not the same as value val.  It could be some value as long as it was a different value?  The comment could be interpreted that the CHECK was to make certain ret was not any value.

Reply:
I wasn't looking at comments - just warnings. We should a spellchecker if we want perfect spelling.

Reply:
Ha..ha...  44 of them!  I think it should be "If it is,".  I'll fix them after you merge and I check out to go through the CHECK/VERIFY macros as we agreed before.